docs/configure/upgrading.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Upgrading Storybook'
+---
+
+The frontend ecosystem is a fast-moving place. Regular dependency upgrades are way of life, whether it's upgrading a framework, library, tooling, or all of the above! Storybook provides a few resources to help ease the pain of upgrading.
+
+## Upgrade script
+
+The most common upgrade is Storybook itself. Storybook follows [Semantic Versioning](https://semver.org/). We release patch releases with bugfixes continuously, minor versions of Storybook with new features every few months, and major versions of Storybook with breaking changes roughly once per year.
+
+To help ease the pain of keeping Storybook up-to-date, we provide a command-line script:
+
+```sh
+npx sb@next upgrade
+```
+
+This upgrades all your Storybook packages to the latest stable version, perform sanity checks of your package versions, and also check for [automigrations](#automigrate) to automatically update your configuration.
```

Comment:
I just pushed a commit with edits to this page, but we should avoid using the term "sanity check": https://www.selfdefined.app/definitions/sanity-check/

Reply:
🙀 thank you! 🙀